OpenRA.Game/FileFormats/VocLoader.cs

```diff
@@ -0,0 +1,348 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OpenRA.FileFormats
+{
+	public class VocLoader : Stream
```

Comment:
Can we drop the stream stuff for now?  It's going to be a while before we can support that properly, and these sound loaders are soon going to be hidden behind an interface where nobody can see this.